src/Haskell/Ide/Engine/Plugin/Liquid.hs

```diff
@@ -4,8 +4,10 @@
 {-# LANGUAGE NamedFieldPuns      #-}
 module Haskell.Ide.Engine.Plugin.Liquid where
 
+import           Control.Concurrent.Async
 import           Control.Monad
 import           Control.Monad.IO.Class
+import           Control.Exception (bracket)
 #if __GLASGOW_HASKELL__ < 804
```

Comment:
This line can now be removed

Reply:
ok

Reply:
Except it fails -Wall on GHC 8.4.3 if the #if is removed